### PR TITLE
DDF-3284 - Removed the UI button for exporting configuration settings

### PR DESCRIPTION
**Pull request by paouelle**
#### What does this PR do?
This PR removes the ability to perform a migration export of all configuration settings from the UI.
#### Who is reviewing it? 
@tbatie 
@djblue 
@brjeter 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Choose 2 committers to review/merge the PR. 
(please choose ONLY two committers from below, delete the rest)
@figliold
@ricklarsen - Documentation
#### How should this be tested? (List steps with links to updated documentation)
Open the UI and make sure the button is no longer there to trigger the export. Quickly verify a migration export from the command line still works.
#### What are the relevant tickets?
[DDF-3284](https://codice.atlassian.net/browse/DDF-3284)
#### Checklist:
- [x] Documentation Updated
- [x] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: platform/admin/ui/src/main/webapp/js/views/module/ModuleDetail.layout.js [hunk 1]
```diff
@@ -40,13 +39,8 @@ define([
             'click .header > .btn': 'export'
```
Comment (djblue) [inline platform/admin/ui/src/main/webapp/js/views/module/ModuleDetail.layout.js]: I think we can remove this click event handler as well.

**Comment by paouelle** [comment]
build now

**Comment by paouelle** [comment]
build now

**Comment by tbatie** [comment]
Hero'd successfully. Confirmed export still works and export button no longer exists

**Pull request merged**